src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -276,9 +277,7 @@ static String createQuery(Map<String, Object> params)
   private static List<Parameter> flattenParamsList(List<Object> params, String keyPrefix)
       throws InvalidRequestException {
     List<Parameter> flatParams = new ArrayList<Parameter>();
-    Iterator<?> it = ((List<?>) params).iterator();
-    String newPrefix = String.format("%s[]", keyPrefix);
-
+    ListIterator<?> it = ((List<?>) params).listIterator();
     // Because application/x-www-form-urlencoded cannot represent an empty
     // list, convention is to take the list parameter and just set it to an
     // empty string. (e.g. A regular list might look like `a[]=1&b[]=2`.
```

Comment:
Would you mind updating this comment to something like this (now that we expect to have integer indexes everywhere):

``` java
	// Because application/x-www-form-urlencoded cannot represent an empty
	// list, convention is to take the list parameter and just set it to an
	// empty string. (e.g. A regular list might look like `a[]=1&b[]=2` or
	// `a[0]=1&b[1]=2` when encoded with Stripe's style. Emptying it would look
	// like `a=`.)
```

Reply:
There's also a duplicate comment in `flattenParamsArray` that should be updated.

Reply:
fixed!